### DISCUSSION TITLE
Symfonisk controller & media hook integration

### DISCUSSION THREAD

**Discussion by jurgen2005** [EPMatt/awesome-ha-blueprints#393] [Q&A]
Hi, for some reason my Symfonisk controller & media hook integration only pauses and starts the music.
zha_events shows that rotary action is detected properly.
Any ideas to what is wrong?

**Comment by Andyliga** [comment]
The solution is listed here as a "fix" but apparently the code hasn't been updated:
https://github.com/EPMatt/awesome-ha-blueprints/pull/314#issuecomment-1114282524

This will 100% fix the issue with the controller.
Don't forget to reload the automations, from Developer Tools, after modifying the blueprint in "/config".
